src/wp-includes/block-template-utils.php

```diff
@@ -206,6 +206,33 @@ function get_default_block_template_types() {
 		),
 	);
 
+	// Add a title and description to post format templates.
+	$post_formats = array(
+		'aside',
+		'audio',
+		'chat',
+		'gallery',
+		'image',
+		'link',
+		'quote',
+		'status',
+		'video',
+	);
```

Comment:
Actually, it may be possible to use `get_post_format_strings` instead of this custom `$post_formats` array.